cfme/automate/__init__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from navmazing import NavigateToSibling
+from widgetastic.widget import View
+from widgetastic_patternfly import Accordion, Dropdown
+
+from cfme import BaseLoggedInPage
+from utils.appliance import IPAppliance
+from utils.appliance.endpoints.ui import navigator, CFMENavigateStep
+
+
+class AutomateCustomizationView(BaseLoggedInPage):
+    @property
+    def is_displayed(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Automate', 'Customization'])
+
+    @View.nested
+    class provisioning_dialogs(Accordion):  # noqa
+        ACCORDION_NAME = 'Provisioning Dialogs'
+        CONTAINS_TREE = True
+
+    @View.nested
+    class service_dialogs(Accordion):  # noqa
+        ACCORDION_NAME = 'Service Dialogs'
+        CONTAINS_TREE = True
+
+    @View.nested
+    class buttons(Accordion):  # noqa
+        CONTAINS_TREE = True
+
+    @View.nested
+    class import_export(Accordion):  # noqa
+        ACCORDION_NAME = 'Import/Export'
+        CONTAINS_TREE = True
+
```

Comment:
So, for this case, we will not want to use infobox as that one is less capable than the SummaryMixin we have now. I think I will try converting the Summary into a Widget which will allow us to continue using that in similar manner. I will probably not make it so one object encompasses all of the table boxes with data, but I might modify it so each widget is one table-box :) @mshriver what do you say on that?